### PR TITLE
Enhancements for generate-docs

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/44063

## Description
Added the following changes:

1. Supports giving filepath and comma-seperated list as command examples to `generate-docs` to scripts and integrations
2. Support for giving more than one example to same command or script

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 1]
```diff
@@ -454,73 +454,68 @@ def disable_md_autolinks(markdown: str) -> str:
...
-    if context_example and context_example != '{}':
+    else:
         example.extend([
-            '#### Context Example',
-            '```json',
-            '{}'.format(context_example),
-            '```',
+            '',
+            '#### Command',
+            '```{}```'.format(cmd_from_yaml['name']),
```
Comment (DeanArbel) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:467]: are you sure this part is needed? I don't see it in the previous implementation.
Please show me offline an example output.

File: demisto_sdk/tests/test_files/integration-display-credentials-none/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+Use the IPinfo.io API to get data about an IP address.
```
Comment (DeanArbel) [inline demisto_sdk/tests/test_files/integration-display-credentials-none/README.md:1]: What is this?

**Review by DeanArbel — CHANGES_REQUESTED** [review]

File: demisto_sdk/tests/test_files/integration-display-credentials-none/README.md (hunk 2 above)
Comment (ilaner) [reply demisto_sdk/tests/test_files/integration-display-credentials-none/README.md]: have no idea deleted it

**Review by DeanArbel — APPROVED** [review]
@ilaner Please look at the validations.

File: demisto_sdk/commands/generate_docs/README.md [hunk 3]
```diff
@@ -14,8 +14,9 @@ For integration - Top use-cases. Number the steps by '*' (i.e. '\* foo. * bar.')
 * **-c, --command**
 A comma-separated command names to generate doc for, will ignore the rest of the commands. e.g xdr-get-incidents,xdr-update-incident
 * **-e, --examples**
-Integrations: path for file containing command examples. Each command should be in a separate line.
-  Scripts: the script example surrounded by quotes. For example: `-e '!ConvertFile entry_id=<entry_id>'`
+- Path for file containing examples. Each command should be in a separate line. or a comma seperated list of commands
+- Comma seperated list of the example, surrounded by quotes.
+If the file or list containing a command with more than one example for it, all of them will be used.
```
Comment (DeanArbel) [inline demisto_sdk/commands/generate_docs/README.md:19]:
```suggestion
- Path for file containing examples. Each command should be in a separate line. or a comma seperated list of commands
- Comma separated list of examples, wrapped by quotes.
If the file or list contains a command with more than one example, all of them will be used.
```

**Pull request merged**